test/fixtures/shared/terraform.tfvars

```diff
@@ -0,0 +1,8 @@
+project_id = "ci-vm-module-b7dc"
```

Comment:
we shouldn't check this in

Reply:
I have removed all the terraform.tfvars / terraformform.tfvars.example and symbolic links. I have removed the 'Running Test' and 'Running Test' sections from the main Readme.md. The documentation was irrelevant. Running the tests is fully covert by the CONTRIBUTION.md